cli/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "shift-cli",
   "description": "ShiftJS CLI Tool",
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
This is probably wrong, if there is a `rush change` of type `patch` it will automatically increment the version to `0.1.3` on publish.